integration/service/create_test.go

```diff
@@ -309,6 +312,144 @@ func TestCreateServiceConfigFileMode(t *testing.T) {
 	assert.NilError(t, err)
 }
 
+// TestServiceCreateSysctls tests that a service created with sysctl options in
+// the ContainerSpec correctly applies those options.
+//
+// To test this, we're going to create a service with the sysctl option
+//
+//   {"net.ipv4.ip_nonlocal_bind": "0"}
+//
+// which is going to run a command to cat the value of that parameter into the
+// service logs, and we'll verify that it is 0 as expected.
+//
+// Next, we'll remove that service and create a new service with that option
+// set to 1. This means that no matter what the default is, we can be confident
+// that the sysctl option is applying as intended.
+//
+// Additionally, we'll do a service inspect to verify that the inspect output
+// includes the desired sysctl option.
+//
+// We're using net.ipv4.ip_nonlocal_bind because it's something that I'm fairly
+// confident won't be modified by the container runtime, and won't blow
+// anything up in the test environment
+func TestCreateServiceSysctls(t *testing.T) {
+	defer setupTest(t)()
```

Comment:
Can you add a skip for older daemon versions? (assuming this is planned for inclusion in API 1.39)

```go
	skip.If(t, versions.LessThan(testEnv.DaemonAPIVersion(), "1.39"), "feature was added in API version 1.39")
```

Reply:
Do we need a separate test for `service update` ? (perhaps overkill)

Reply:
Yes, we should make one, actually, to verify that `service update` goes through correctly when Sysctls are changed.